libraries/ESP8266HTTPClient/examples/BasicHttpClient/BasicHttpClient.ino

```diff
@@ -40,34 +42,39 @@ void loop() {
   // wait for WiFi connection
   if ((WiFiMulti.run() == WL_CONNECTED)) {
 
+    WiFiClient client;
+
     HTTPClient http;
 
     USE_SERIAL.print("[HTTP] begin...\n");
     // configure traged server and url
     //http.begin("https://192.168.1.12/test.html", "7a 9c f4 db 40 d3 62 5a 6e 21 bc 5c cc 66 c8 3e a1 45 59 38"); //HTTPS
```

Comment:
let's remove the comment, maybe it's better to add "BasicHTTPSClient.ino" instead.

Reply:
Removing comment and adding a BasicHTTPSClient.ino with fingerprint security check